src/components/Avatar/Avatar.tsx

```diff
@@ -16,7 +16,7 @@ enum Status {
   Errored = 'ERRORED',
 }
 
-const STYLE_CLASSES = ['one', 'two', 'three', 'four', 'five', 'six'];
+const STYLE_CLASSES = ['one', 'two', 'three', 'four', 'five'];
```

Comment:
This will cause visual regressions, as indicated by the failing Percy test.

We can remove this class conditionally based on the `useFeatures` hook returning `globalTheming === true`, which will preserve backwards compatibility.

Reply:
Oh nice, I'll make the change in a bit 👍